src/content/self-hosted/install/crds.mdx

```diff
@@ -0,0 +1,61 @@
+
+If you prefer to define the registries directly in your cluster, you can do it creating an [Opaque Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret) in the cluster with the data needed for the registry. Depending on the type of registry, you will need different fields. Once the secret is created, you will need to create a resource with the apiVersion `admin.okteto.com/v1`, which is one of the CRDs installed by Okteto, refering to the secret which contains the credentials.
+
+In that way, if you want to define static credentials (username and password) for a private registry, you just need to create a secret in the Okteto namespace with the user and pass to be used. After that, you have to define a CRD with some information. The name of the resource has to be the hostname of the registry, so imagine you want to add a configuration for docker's registry, you will need to create a resource like this:
+
+```yaml
+apiVersion: admin.okteto.com/v1
+kind: Registry
+metadata:
+  name: index.docker.io # hostname of the registry to be configured
+  namespace: okteto
+spec:
+  static:
+    passwordSecretRef:
+      key: <field in referenced secret> # key of the secret which contains the password
+      name: <secret-name> # name of the secret where the password is stored
+    userSecretRef:
+      key: <field in referenced secret> # key of the secret which contains the username
+      name: <secret-name> # name of the secret where the username is stored
+```
+
+In the case of AWS, where you have to specify the Access Key and Access Secret, the process is similar to the static credentials one. You will need to create a secret with those values, and then, create a CRD like this (the name of the resource has to be the hostname of the registry to be configured and it has to end with `.amazonaws.com`):
+
+```yaml
+apiVersion: admin.okteto.com/v1
+kind: Registry
+metadata:
+  name: 111122223333.dkr.ecr.eu-central-1.amazonaws.com # hostname of the registry to be configured
+  namespace: okteto
+spec:
+  awsIamUser:
+    accessKeyIDSecretRef:
+      key: <field in referenced secret> # key of the secret which contains the access key
+      name: <secret-name> # name of the secret where the access key is stored
+    secretAccessKeySecretRef:
+      key: <field in referenced secret> # key of the secret which contains the access secret key
+      name: <secret-name> # name of the secret where the access secret is stored
+```
+
+> Be aware that any change you do in those credentials might take a few minutes to be propagated to all the components in the cluster. To make sure Okteto is reading properly those registries, you can check it going to the [Admin dashboard](administration/dashboard.mdx/#registry-credentials). They will be displayed, but they can't be modified from the UI. If you want to manage them from the UI, they have to be created from there.
```

Comment:
```suggestion
:::note
Any changes you make to credentials can take a few minutes to propagate to all components in the cluster. To make sure Okteto is able to access the registries, you can check in the [Admin dashboard](administration/dashboard.mdx/#registry-credentials). They will be displayed, but they can't be modified from the UI. If you want to manage them from the UI, they have to be created from there.
:::
```

Is the last sentence still valid with these latest changes? I assume it may be accurate and specific to AWS but wanted to double check.

I may have further suggestions on this copy depending on the clarification here 😄

Reply:
This note is for both types (static and AWS) and it is still valid. If you create the credentials in this way (using CRDs), you will only see them in the UI but you cannot modify them. If you want to be able to edit them from the UI, you would need to create them from there

Reply:
Thanks for the clarification 🙇 

I updated my proposed change based on that but let me know if you disagree with it

Reply:
Your proposal looks good, I just commit it